searx/engines/qwant.py

```diff
@@ -76,38 +106,78 @@ def request(query, params):
     if not query:
         return None
 
-    count = 10  # web: count must be equal to 10
+    q_locale = traits.get_region(params["searxng_locale"], default='en_US')
+
+    url = api_url + f'{qwant_categ}?'
+    args = {'q': query}
+    params['raise_for_httperror'] = False
+
+    if qwant_categ == 'web-lite':
+
+        # qwant-lite delivers only 5 pages maximum
+        if params['pageno'] > 5:
+            return None
+
+        url = web_lite_url + '?'
+        args['locale'] = q_locale.lower()
+        args['l'] = q_locale.split('_')[0]
+        args['s'] = params['safesearch']
+        args['p'] = params['pageno']
+
+        params['raise_for_httperror'] = True
 
-    if qwant_categ == 'images':
-        count = 50
-        offset = (params['pageno'] - 1) * count
+    elif qwant_categ == 'images':
+
+        args['locale'] = q_locale
+        args['safesearch'] = params['safesearch']
+
+        args['count'] = 50
+        offset = (params['pageno'] - 1) * args['count']
         # count + offset must be lower than 250
-        offset = min(offset, 199)
-    else:
-        offset = (params['pageno'] - 1) * count
-        # count + offset must be lower than 50
-        offset = min(offset, 40)
+        args['offset'] = min(offset, 199)
 
-    params['url'] = url.format(
-        keyword=qwant_categ,
-        query=urlencode({'q': query}),
-        offset=offset,
-        count=count,
-    )
+    else:  # web, news, videos
 
-    # add quant's locale
-    q_locale = traits.get_region(params["searxng_locale"], default='en_US')
-    params['url'] += '&locale=' + q_locale
+        args['locale'] = q_locale
+        args['safesearch'] = params['safesearch']
 
-    # add safesearch option
-    params['url'] += safe_search_map.get(params['safesearch'], '')
+        args['count'] = 10
+        offset = (params['pageno'] - 1) * args['count']
+        # count + offset must be lower than 50
+        args['offset'] = min(offset, 39)
+
+    params['url'] = url + urlencode(args)
 
-    params['raise_for_httperror'] = False
     return params
 
 
 def response(resp):
-    """Get response from Qwant's search request"""
+
+    if qwant_categ == 'web-lite':
+        return parse_web_lite(resp)
+    return parse_web_api
```

Comment:
I I have not seen this typo before I merged this PR / typo is fixed master branch in 7398d525c